plugins/octant/cmd/antrea-octant-plugin/traceflow.go

```diff
@@ -383,6 +334,273 @@ func (p *antreaOctantPlugin) actionHandler(request *service.ActionRequest) error
 	}
 }
 
+func checkNamespace(request *service.ActionRequest) (string, error) {
+	srcNamespace, err := request.Payload.String(srcNamespaceCol)
+	if err != nil {
+		alertPrinter(request, invalidInputMsg+"failed to get srcNamespace as string",
+			"Failed to get source Namespace as string", nil, err)
+		return "", err
+	}
+	return srcNamespace, nil
+}
+
+func checkDstType(request *service.ActionRequest) (string, error) {
+	dstType, err := request.Payload.StringSlice(dstTypeCol)
+	if err != nil || len(dstType) == 0 {
+		alertPrinter(request, invalidInputMsg+"failed to get dstType as string slice:",
```

Comment:
I'm a little confused about the style of the log message: some messages have trailing ":" while some don't, some have a trailing space while some done.
And all wrapped messages should not be capitalized like here, but they are capitalized in some places.

Reply:
I refined the log print via a shared method but keep the original data,  I will revisit those log and make they consistent.

Reply:
and for the second message, it's an alert message which will be printed in UI, I suppose it should be capitalized?

Reply:
some of error messages are just printed directly due to errs and err are both nil, so no need to have trailing ":"

Reply:
I refined the alertPrinter and remove the trailing ":" for most cases.